_inc/client/security/sso.jsx

```diff
@@ -74,7 +74,7 @@ export const SSO = withModuleSettingsFormHelpers(
 					>
 						<p>
 							{ __(
-								'Add an extra layer of security to your website by enabling WordPress.com log in and secure ' +
+								'Add an extra layer of security to your website by enabling WordPress.com login and secure ' +
```

Comment:
I don't know the word for it, but I think here it's not a noun "WordPress log in and authentication"

Reply:
I'll let @obi2020 weigh in on this.

Reply:
OK, let's consider it a noun, works for me :)